README.md

```diff
@@ -19,7 +22,6 @@ useful for Rīchi Mahjong players.
 ## Quickstart
 ```
 pip install -r requirements.txt
-cp kccbadgeserver/settings/{development_settings.py.example,local_settings.py}
```

Comment:
lol we still had the old project name in the README and nobody noticed for 3 years